src/app/grid-row-pinning/grid-row-pinning.sample.html

```diff
@@ -1,22 +1,22 @@
 <div class="wrapper">
-    <app-page-header title="Grid Column Pinning">
+    <app-page-header title="Grid Row Pinning">
         Allows rows to be pinned to the beginning/end of the grid.
     </app-page-header>
-    <button (click)="grid1.addRow({'ID': 'TEST', 'CompanyName': 'Test'})">Add Row</button>
-    <button (click)="exportButtonHandler()">Export</button>
     <div class="sample-content">
         <div class="sample-column">
             <div class="sample-buttons">
                 <button igxButton="raised" (click)="saveGridState()">Save state</button>
                 <button igxButton="raised" (click)="restoreGridState()">Restore state</button>
+                <button igxButton="raised" (click)="grid1.addRow({'ID': 'TEST', 'CompanyName': 'Test'})">Add Row</button>
+                <button igxButton="raised" (click)="exportButtonHandler()">Export</button>
             </div>
             <div class="sample-switches">
                 <igx-switch (change)='onRowChange()' style="padding-left: 10px"> Bottom Row Pinning toggle</igx-switch>
                 <igx-switch (change)='onChange()' style="padding-left: 10px"> Right Column Pinning toggle</igx-switch>
             </div>
             <igx-grid [igxGridState]="options" [allowFiltering]='true' [primaryKey]='"ID"' [pinning]="pinningConfig"
-                [columnHiding]='true' [showToolbar]='true' [columnPinning]='true' #grid1 [data]="data" [width]="'800px'"
-                [height]="'500px'" [rowSelectable]="false">
+                [columnHiding]='true' [showToolbar]='true' [columnPinning]='true' #grid1 [data]="data" [width]="'1200px'"
```

Comment:
@ChronosSF I have the feeling that after this change, 5/6 of the space would still be white space on 1440p monitor...

Reply:
Sure, I tried 100% for 1440p but that means no scrollbar which makes column pinning meaningless. I couldn't find a sample that has a data source with a good amount of columns for a 1440p 100% display and I hope you'll agree that huge columns are also not particularly pretty. 

If we scrap the idea of merging it today I'll find a proper data source for 100% on 1440p but it will take some time.

Reply:
With the nav bar open the grid + navbar take a little bit more than half the horizontal space right now.